README.md

```diff
@@ -94,7 +94,9 @@ cp kubectl-crossplane /usr/local/bin
 #### Install the Platform Configuration
 
 ```console
-PLATFORM_CONFIG=registry.upbound.io/upbound/platform-ref-aws:v0.1.0
+# Check the latest version available in https://github.com/upbound/platform-ref-aws/releases
```

Comment:
Nitpick: I assume all releases will to the registry immediately, but I don't think we currently have this automated, so it might make sense to just check for latest in registry rather than on github

Reply:
Yeah, pointing to registry is more precise. Updated.